demos/python_demos/asl_recognition_demo/README.md

```diff
@@ -0,0 +1,85 @@
+# ASL Recognition Python* Demo
+
+This demo demonstrates how to run ASL (American Sign Language) Recognition models using OpenVINO&trade; toolkit.
+
+## How It Works
+
+The demo application expects an ASL recognition model in the Intermediate Representation (IR) format.
+
+As input, the demo application takes:
+* a path to a video file or a device node of a web-camera specified with a command line argument `--input`
+* a path to a file in `JSON` format with ASL class names `--class_map`
+
+The demo workflow is the following:
+
+1. The demo application reads video frames one by one, runs person detector that extracts ROI, tracks the ROI of very first person. Additional process is used to prepare the bach of frames with constant framerate.
+2. Batch of frames and extracted ROI are passed to artificial neural network that predicts the ASL gesture.
+3. The app visualizes results of it work as graphical window where following objects are shown:
```

Comment:
```suggestion
3. The app visualizes results of it works as graphical window where following objects are shown:
```

Reply:
It breaks the sentence. Changed to `its work`.

Reply:
You are right